migrations/20210624_confirmations.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE conn_confirmations (
+  confirmation_id BLOB NOT NULL,
+  conn_alias BLOB NOT NULL,
+  sender_key BLOB NOT NULL,
+  sender_conn_info BLOB NOT NULL,
+  approved INTEGER NOT NULL,
+  own_conn_info BLOB,
+  created_at TEXT NOT NULL DEFAULT (datetime('now')),
+  PRIMARY KEY (confirmation_id),
+  FOREIGN KEY (conn_alias)
```

Comment:
Suggest we stop separating field indexes from fields, when possible, it only increases cognitive load, doesn't solve any problem.